libs/ui/tooltip/brain/src/lib/brn-tooltip-content.component.ts

```diff
@@ -219,6 +219,11 @@ export class BrnTooltipContentComponent implements OnDestroy {
 		const tooltip = this._tooltip?.nativeElement;
 		if (!tooltip || !this._isBrowser) return;
 		this._renderer2.setStyle(tooltip, 'visibility', isVisible ? 'visible' : 'hidden');
+		if (isVisible){
+			this._renderer2.removeStyle(tooltip, 'display');
+		}else{
+			this._renderer2.setStyle(tooltip, 'display', 'none');
+		}
```

Comment:
This seems unrelated, please rebase from the `main` branch!

Reply:
@ajitzero this is my first time working on an open source project. You're right, I'm not sure why this commit is here, and I don't know how to fix it. I tried to rebase from the main branch, but it doesn't seem to work; I'm surely doing something wrong. Feel free to delete this PR, and I'll delete my forked repo and create a new, clean PR. Alternatively, if you prefer, you can create a new one yourself.

Reply:
No worries! git can get confusing with forks and whatnot. I see you have 3 extra commits in this PR https://github.com/goetzrobin/spartan/pull/342/commits - I'm not sure what could have caused this either. Could you please try removing these last three commits?

Try these CLI commands:
```shell
# go to your repo/fork's branch with the changes
git checkout main

# remove the last N (here, 3) commits: git reset --hard HEAD~N
git reset --hard HEAD~3

# Update your fork on GitHub forcibly and overwrite it.
# We don't do this for actual projects, but is common for cleaning up PRs.
git push --force
```
To be cautious though, I would suggest running the reset command without the `--hard` flag. This will only "undo" the changes and keep them available locally. You can then decide what to skip manually.